### PR TITLE
Content Pod Group, Content Panel, Hero

### PR DESCRIPTION
**Pull request by cotoole-merkle**

### CODE REVIEW

**Review by cheeto-bandito — CHANGES_REQUESTED** [review]
This should be moved into a Draft state until we have group consensus on Base Template standards.

File: scaffolding/merkle.feature.basiccontent/src/Feature/BasicContent/items/Templates/Basic Content/ContentPodGroup.yml
Comment (JRondeau16) [inline scaffolding/merkle.feature.basiccontent/src/Feature/BasicContent/items/Templates/Basic Content/ContentPodGroup.yml:1]: Can you add an icon to this so it looks nice and pretty, please?

File: scaffolding/merkle.feature.basiccontent/src/Feature/BasicContent/items/Templates/Basic Content/Hero.yml
Comment (JRondeau16) [inline scaffolding/merkle.feature.basiccontent/src/Feature/BasicContent/items/Templates/Basic Content/Hero.yml:1]: Fields look good based on what's currently in the contract but Ryan has some feedback that might affect what fields are on this.

Comment (JRondeau16) [inline scaffolding/merkle.feature.basiccontent/src/Feature/BasicContent/items/Templates/Basic Content/Hero.yml:1]: Can you add an icon to this so it looks nice and pretty, please?

File: scaffolding/merkle.feature.basiccontent/src/Feature/BasicContent/items/Templates/Basic Content/ContentPanel.yml
Comment (JRondeau16) [inline scaffolding/merkle.feature.basiccontent/src/Feature/BasicContent/items/Templates/Basic Content/ContentPanel.yml:1]: Can you add an icon to this so it looks nice and pretty, please?

File: scaffolding/merkle.feature.basiccontent/src/Feature/BasicContent/items/Renderings/Basic Content/ContentPanel.yml
Comment (JRondeau16) [inline scaffolding/merkle.feature.basiccontent/src/Feature/BasicContent/items/Renderings/Basic Content/ContentPanel.yml:1]: I noticed the datasource location and Rendering Contents Resolver are not set. Could you set those and create any associated templates (I looked on Focus and I see things like datasource folders)?

File: scaffolding/merkle.feature.basiccontent/src/Feature/BasicContent/items/Renderings/Basic Content/Hero.yml
Comment (JRondeau16) [inline scaffolding/merkle.feature.basiccontent/src/Feature/BasicContent/items/Renderings/Basic Content/Hero.yml:1]: I noticed the datasource location and Rendering Contents Resolver are not set. Could you set those and create any associated templates (I looked on Focus and I see things like datasource folders)?

File: scaffolding/merkle.feature.basiccontent/src/Feature/BasicContent/items/Renderings/Basic Content/ContentPodGroup.yml
Comment (JRondeau16) [inline scaffolding/merkle.feature.basiccontent/src/Feature/BasicContent/items/Renderings/Basic Content/ContentPodGroup.yml:1]: We'll also need a Content Pod rendering that goes inside a content pod group via placeholder. Content Pod will have the datasource template, etc. filled out. Content Pod Group is just a container with a placeholder and no associated datasource template.

**Review by JRondeau16 — COMMENTED** [review]
There is a separate ticket for the removal of foundation base templates https://degdigital.atlassian.net/browse/GASA-76 so I'm good with this being merged in with existing references to foundation base templates or combining GASA-76 into this. Your choice, @cotoole-merkle

**Comment by JRondeau16** [comment]
image

Please set a display name to make these content author friendly.

File: scaffolding/merkle.feature.basiccontent/src/Feature/BasicContent/items/Renderings/Basic Content/ContentPanel.yml
Comment (JRondeau16) [reply scaffolding/merkle.feature.basiccontent/src/Feature/BasicContent/items/Renderings/Basic Content/ContentPanel.yml:1]: Looks good now.

File: scaffolding/merkle.feature.basiccontent/src/Feature/BasicContent/items/Renderings/Basic Content/Hero.yml
Comment (JRondeau16) [reply scaffolding/merkle.feature.basiccontent/src/Feature/BasicContent/items/Renderings/Basic Content/Hero.yml:1]: Looks good.

File: scaffolding/merkle.feature.basiccontent/src/Feature/BasicContent/items/Templates/Basic Content/ContentPanel.yml
Comment (JRondeau16) [reply scaffolding/merkle.feature.basiccontent/src/Feature/BasicContent/items/Templates/Basic Content/ContentPanel.yml:1]: Looks gorgeous!

File: scaffolding/merkle.feature.basiccontent/src/Feature/BasicContent/items/Templates/Basic Content/Hero.yml
Comment (JRondeau16) [reply scaffolding/merkle.feature.basiccontent/src/Feature/BasicContent/items/Templates/Basic Content/Hero.yml:1]: Looks gorgeous!

File: scaffolding/merkle.feature.basiccontent/src/Feature/BasicContent/items/Templates/Basic Content/ContentPodGroup.yml
Comment (JRondeau16) [reply scaffolding/merkle.feature.basiccontent/src/Feature/BasicContent/items/Templates/Basic Content/ContentPodGroup.yml:1]: Looks gorgeous!

**Review by JRondeau16 — CHANGES_REQUESTED** [review]
Add placeholder(s) as needed for content pod group so content pods can go inside of it.

**Review by JRondeau16 — APPROVED** [review]
I haven't seen many things in my life as beautiful as this completed PR. Approved!

**Comment by JRondeau16** [comment]
> This should be moved into a Draft state until we have group consensus on Base Template standards.

We're handle the base template removal in a separate ticket. Giving the okay to merge this PR.

**Pull request merged**